packages/tokens/vite.config.ts

```diff
@@ -0,0 +1,37 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { resolve } from 'node:path';
+
+import react from '@vitejs/plugin-react';
+import { defineConfig } from 'vite';
+import dts from 'vite-plugin-dts';
+
+export default defineConfig({
+    plugins: [
+        react(),
+        dts({
+            rollupTypes: true,
+            include: ['src/components/ThemeProvider'],
+            copyDtsFiles: true,
+        }),
+    ],
+    build: {
+        outDir: resolve(__dirname, 'dist/theme'),
+        lib: {
+            entry: resolve(__dirname, 'src/components/ThemeProvider/ThemeProvider.tsx'), // Your entry file
+            name: 'FondueThemeProvider', // Global variable name when used in browser
```

Comment:
Let's make a correct entrypoint
```suggestion
            entry: resolve(__dirname, 'src/index.ts'),
            name: 'FondueTokens',
```